### DISCUSSION TITLE
Add content / blocks

### DISCUSSION THREAD

**Discussion by al1p** [ultimate-notion/ultimate-notion#29] [General]
Hi, thanks for this great work ! Most of the operations are intuitive and efficient. I am stuck with the inability to add content / blocks. I'm thinking of using [notion-sdk-py](https://github.com/ramnes/notion-sdk-py) as a work around. Is there any plan / need for help for this important feature ?

**Comment by FlorianWilhelm** [comment]
@al1p : Sorry for the late response. I'm working currently on the PR #35 with tons of updates and the ability to add content to a page. Stay tuned as it will be released in the next couple of weeks. I would love to get some feedback from you then 🙇

**Comment by al1p** [comment]
Looking forward to it ! Thanks !

**Reply by FlorianWilhelm** [reply]
Just released version 0.5 with [new docs about creating the content of a page](https://ultimate-notion.com/latest/usage/page_advanced/). Let me know what you think.

**Reply by al1p** [reply]
Great, but I've been stuck with this bug : https://github.com/ultimate-notion/ultimate-notion/issues/39

**Reply by FlorianWilhelm** [reply]
Fixed it. Any other problems you came across?

**Comment by al1p** [comment]
I'm playing the new page update capabilities, it works very smoothly ! that's great work 👏  

Thanks a lot.

So nice to be able to update content in such a straigthforward manner.
